proto/noble/forwarding/v1/packet.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package noble.forwarding.v1;
+
+option go_package = "github.com/noble-assets/noble/v5/x/forwarding/types";
+
+message RegisterAccountData {
+  string recipient = 1;
+}
+
+message RegisterAccountMemo {
+  message RegisterAccountDataWrapper {
+    RegisterAccountData forwarding = 1;
+  }
+
+  RegisterAccountDataWrapper noble = 1;
+}
```

Comment:
Is there a reason the wrapper is needed here?

Reply:
@agouin The reason for the wrapper is so the JSON schema looks like this:

```json
{
  "noble": {
    "forwarding": {
      "recipient": "cosmos1blah"
    }
  }
}
```

This is because Packet Forward Middleware already uses the `forward` domain, which would confuse users.